@pauliescanlon/gatsby-mdx-embed/src/index.js

```diff
@@ -11,6 +11,7 @@ export {
   PinterestBoard,
   PinterestFollowButton
 } from './components/Pinterest'
+export { SimplecastEpisode as SimplecastEpesode } from './components/Simplecast'
```

Comment:
do we need this can you just export `SimplecastEpisode` from ...

Reply:
No, we don't, I missed it!

Fixed, I'll push the changes.